src/sidebar/services/test/frame-sync-test.js

```diff
@@ -315,18 +348,27 @@ describe('FrameSyncService', () => {
       frameSync.connect();
     });
 
-    it('sends a "publicAnnotationCountChanged" message to the frame when there are public annotations', () => {
+    it('sends a "publicAnnotationCountChanged" message to the frame when there are public annotations', async () => {
+      await connectGuest();
+      emitGuestEvent('documentInfoChanged', fixtures.htmlDocumentInfo);
+      fakeStore.frames()[0].isAnnotationFetchComplete = true;
```

Comment:
This step should have been present previously, but the test could get away without it due to the way that `store.frames()` was faked in the tests.

Reply:
> isAnnotationFetchComplete is already set it to true by the mocking logic. 

Indeed. This was left over from an earlier revision by mistake.